try:
    raise ValueError
else:
    print('here')
